test/e2e/common/pods.go

```diff
@@ -870,6 +875,154 @@ var _ = framework.KubeDescribe("Pods", func() {
 		err = wait.PollImmediate(podRetryPeriod, podRetryTimeout, checkPodListQuantity(f, "type=Testing", 0))
 		framework.ExpectNoError(err, "found a pod(s)")
 	})
+
+	ginkgo.It("should run through the lifecycle of Pods and PodStatus", func() {
```

Comment:
This test is flaking. See https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/93398/pull-kubernetes-e2e-gce-ubuntu-containerd/1286493826562658304/ as an example